src/test/java/by/issoft/tests/CreateAccountTest.java

```diff
@@ -0,0 +1,48 @@
+package by.issoft.tests;
+
+import by.issoft.pagefactory.AccountCreationPage;
+import by.issoft.pagefactory.LoginPage;
+import by.issoft.utils.MyTestWatcher;
+import by.issoft.utils.TestBase;
+import com.github.javafaker.Faker;
+import io.qameta.allure.Description;
+import io.qameta.allure.Step;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(MyTestWatcher.class)
+public class CreateAccountTest extends TestBase {
+
+    Faker faker = new Faker();
+    String firstName = faker.name().firstName(),
+            lastName = faker.name().lastName(),
+            email = faker.internet().emailAddress(),
+            password = "TestPass1!",
+            mobileNumber = faker.numerify("##########"),
+            address = faker.address().fullAddress(),
+            city = faker.address().city(),
+            postalCode = mobileNumber = faker.numerify("#####"),
+            state = "Florida";
```

Comment:
I don't see the reason to have those variables. You can do the same when you call methods. 
Like 
.enterLastName(faker.name().lastName())